hugegraph-llm/src/hugegraph_llm/api/models/rag_requests.py

```diff
@@ -70,3 +70,7 @@ class RerankerConfigRequest(BaseModel):
     reranker_type: str
     api_key: str
     cohere_base_url: Optional[str] = None
+
+class LogStreamRequest(BaseModel):
+    log_auth_key: Optional[str] = None
```

Comment:
```suggestion
    log_token: Optional[str] = None
```

maybe `token` is enough?